model_training/cmd_instruction.md

```diff
@@ -12,25 +26,25 @@ PROM_URL= # e.g., http://localhost:9090
 COLLECT_ID= # any unique id e.g., machine name
 ```
 
-### 1.1. by defining start time and end time
+### 1.1. By defining start time and end time
 
 ```bash
 # time value setting
 START_TIME= # format date +%Y-%m-%dT%H:%M:%SZ
 END_TIME= # format date +%Y-%m-%dT%H:%M:%SZ
 
 # query execution
-DATAPATH=/path/to/workspace python cmd/main.py query --benchmark $BENCHMARK --server $PROM_URL --output kepler_query --start-time $START_TIME --end-time $END_TIME --id $COLLECT_ID
+DATAPATH=/path/to/workspace kepler-model query --benchmark $BENCHMARK --server $PROM_URL --output kepler_query --start-time $START_TIME --end-time $END_TIME --id $COLLECT_ID
 ```
 
-### 1.2. by defining last interval from the execution time
+### 1.2. By defining last interval from the execution time
 
 ```bash
 # time value setting
 INTERVAL= # in second
 
 # query execution
-DATAPATH=/path/to/workspace python cmd/main.py query --benchmark $BENCHMARK --server $PROM_URL --output kepler_query --interval $INTERVAL --id $COLLECT_ID
+DATAPATH=/path/to/workspace kepler-model query --benchmark $BENCHMARK --server $PROM_URL --output kepler_query --interval $INTERVAL --id $COLLECT_ID
```

Comment:
can we also remove all references to `cmd/main.py` as well through out the project (in a different PR) ?

Reply:
sure 💯